ocs_ci/deployment/flexy.py

```diff
@@ -252,11 +243,14 @@ def merge_flexy_env(self):
                     config_parser.set('root', ele[0], config.FLEXY[ele[0]])
                 logger.info(f"env updated {ele[0]}:{config.FLEXY[ele[0]]}")
 
-        # write the updated config_parser content back to the env file
-        tmp_file = os.path.join(
-            self.flexy_host_private_conf_dir_path, f"{constants.FLEXY_DEFAULT_ENV_FILE}.tmp"
-        )
-        with open(tmp_file, "w") as fp:
+        # add new keys to env file
+        for key in config.FLEXY:
+            if key not in applied_keys:
+                logger.info(f"Adding {key}={config.FLEXY[key]}")
+                config_parser.set('root', key, f"{config.FLEXY[key]}")
```

Comment:
I'll change the logic of this function - simplify it with just one loop and changing the value or adding new key in one step.

Reply:
Updated by 89a44ee.